### PR TITLE
Allow skip as a value of wp_type

### LINKED ISSUE #1194: Custom Site Template - Allow "skip" as a value for wp_type
## Expected Behavior

I want to be able to quickly scaffold sites with the custom-site-template repo that aren't WordPress based.

## Current Behavior

Not possible, defaults to single site install

## Possible Solution

Allow wp_type to be set to "skip" or "none" or similar.

## Context

Having the database and the nginx conf etc being automatically setup for me is handy. Sometimes I just need to quickly set up an empty site + db. Right now i'm trying to copy a Magento site down to do some local development. I've had to create all of the site setup files manually because there isn't a way to just say "skip wordpress install".

## Your Environment

* VVV version: 2.0.0

### PR DESCRIPTION
**Pull request by LoreleiAurora**
Closes Varying-Vagrant-Vagrants/VVV#1194

Easier to read the diff with this link https://github.com/Varying-Vagrant-Vagrants/custom-site-template/pull/4/files?w=1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Valid values are:
 - single
 - subdomain
 - subdirectory
+- skip
```
Comment (ocean90) [inline README.md]: I think `none` would be a better value for "this is no WP install".

Comment (jeremyfelt) [reply README.md]: +1 for `none` as the config value.

**Comment by tomjn** [comment]
@ocean90 @jeremyfelt I've ammended the PR, can you review?

**Comment by colin-marshall** [comment]
Any progress on this? I would find it useful in VVV. Thanks!

**Comment by tomjn** [comment]
@colin-marshall it's pending testing/review, but there's nothing stopping you from using it, just specify the `branch: Skip-WP` field on a site and it'll clone this branch instead.

**Comment by colin-marshall** [comment]
Thank you @tomjn 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Review by Mte90 — APPROVED** [review]
Tested right now and workds, create a folder empyt without wordpress but with the folder for logs and provision.

**Pull request merged**